### PR TITLE
More categorized data

### PR DESCRIPTION
**Pull request by davidfischer**
This gets us above 50 examples for all our categories.

Replaces https://github.com/readthedocs/ethical-ad-server/pull/640

### CODE REVIEW

File: assets/categorized-data.yml [hunk 1]
```diff
@@ -938,13 +1134,31 @@
...
-- url: "https://modrinth.com/mod/excavated_variants"
+- url: "https://hikari-lightbulb.readthedocs.io/en/latest/guides/application-commands.html"
   topics: []
-- url: "https://modrinth.com/mod/thatorthis"
+- url: "https://google-calendar-simple-api.readthedocs.io/en/latest/events.html"
   topics: []
-- url: "https://modrinth.com/mod/trufflez-arcanum"
+- url: "https://gcovr.com/en/stable/index.html"
+  topics: []  # Testing maybe
+- url: "https://nanogui.readthedocs.io/en/latest/index.html"
```
Comment (ericholscher) [inline assets/categorized-data.yml:1159]: I think this is probably games, right?

**Review by ericholscher — APPROVED** [review]
I have a few nitpicks here, but not sure they're useful. If the model is doing better with this data, let's ship it! 👍

**Comment by davidfischer** [comment]
> I'd be curious to see the model training report in the PR eyes .

I'll post it as soon as I get it =). Capacity issues are real.

**Comment by davidfischer** [comment]
Overall looks a little better with these changes. It isn't huge, but it is meaningful.

## BEFORE THIS CHANGE
I ran this a few times and this is the best result.

```
Training/Test Set Details
================================================================================
Total Training/Test Set Items:      413
Training/Test Set Items for 'datascience':    113
Training/Test Set Items for 'devops':   107
Training/Test Set Items for 'notopic':    63
Training/Test Set Items for 'security':   58
Training/Test Set Items for 'backend':    51
Training/Test Set Items for 'frontend':   40
Training/Test Set Items for 'blockchain':   37


============================= Training pipeline =============================
ℹ Pipeline: ['transformer', 'textcat_multilabel']
ℹ Initial learn rate: 0.0
E    #       LOSS TRANS...  LOSS TEXTC...  CATS_SCORE  SCORE 
---  ------  -------------  -------------  ----------  ------
  0       0           0.00           0.33       48.50    0.48
  1     200           0.31          36.95       83.47    0.83
  3     400           0.28          31.17       89.98    0.90
  5     600           0.19          17.51       88.52    0.89
  6     800           0.21          11.29       92.28    0.92
  8    1000           0.11           7.44       91.29    0.91
 10    1200           0.14          11.39       92.80    0.93
 12    1400           0.12           7.77       91.05    0.91
 13    1600           0.14           4.94       87.92    0.88
 15    1800           0.15           5.16       90.63    0.91
 17    2000           0.16           4.93       90.29    0.90
 18    2200           0.14           5.12       92.03    0.92
 20    2400           0.09           3.66       95.23    0.95
 22    2600           0.05           6.07       93.03    0.93
 24    2800           0.06           3.53       90.75    0.91
 25    3000           0.26           2.77       88.79    0.89
 27    3200           0.10           4.22       89.77    0.90
 29    3400           0.01           3.37       89.55    0.90
 31    3600           0.02           2.82       89.39    0.89
 32    3800           0.01           3.07       90.09    0.90
 34    4000           0.01           2.55       92.74    0.93


================================== Results ==================================

TOK                   100.00
TEXTCAT (macro AUC)   95.23 
SPEED                 16555 


=========================== Textcat F (per label) ===========================

                   P       R       F
datascience    94.74   90.00   92.31
backend       100.00   44.44   61.54
frontend       62.50   50.00   55.56
security       61.11   91.67   73.33
devops         88.89   42.11   57.14
blockchain    100.00   50.00   66.67


======================== Textcat ROC AUC (per label) ========================

              ROC AUC
datascience      0.99
backend          0.95
frontend         0.90
security         0.97
devops           0.90
blockchain       1.00
```

## WITH THESE CHANGES

I only ran it once, but this was the results. 

```
Training/Test Set Details
================================================================================
Total Training/Test Set Items:      518
Training/Test Set Items for 'datascience':    141
Training/Test Set Items for 'devops':   115
Training/Test Set Items for 'notopic':    86
Training/Test Set Items for 'backend':    79
Training/Test Set Items for 'security':   68
Training/Test Set Items for 'frontend':   56
Training/Test Set Items for 'blockchain':   44


============================= Training pipeline =============================
ℹ Pipeline: ['transformer', 'textcat_multilabel']
ℹ Initial learn rate: 0.0
E    #       LOSS TRANS...  LOSS TEXTC...  CATS_SCORE  SCORE 
---  ------  -------------  -------------  ----------  ------
  0       0           0.00           0.32       48.65    0.49
  1     200           0.28          39.64       79.38    0.79
  2     400           0.23          38.68       84.59    0.85
  3     600           0.20          35.12       88.98    0.89
  5     800           0.23          23.42       93.20    0.93
  6    1000           0.51          15.62       93.59    0.94
  7    1200           0.32          15.24       94.22    0.94
  9    1400           4.84          13.31       93.91    0.94
 10    1600           0.29          11.50       89.40    0.89
 11    1800           0.14          14.42       94.06    0.94
 13    2000           0.14           8.83       94.29    0.94
 14    2200           0.20          10.06       91.70    0.92
 15    2400           0.14           7.61       94.73    0.95
 17    2600           0.12           7.02       91.75    0.92
 18    2800           0.13           6.61       93.25    0.93
 19    3000           0.15           5.56       94.52    0.95
 21    3200           0.18           5.06       93.40    0.93
 22    3400           0.03           4.67       94.63    0.95
 23    3600           0.06           3.27       96.39    0.96
 25    3800           0.12           4.71       94.55    0.95
 26    4000           0.06           2.66       95.64    0.96
 27    4200           0.06           1.68       94.33    0.94
 29    4400           0.05           1.67       94.89    0.95
 30    4600           0.14           3.48       93.40    0.93
 31    4800           0.07           3.87       89.10    0.89
 33    5000           0.10           4.95       95.66    0.96
 34    5200           0.06           3.47       92.84    0.93


================================== Results ==================================

TOK                   100.00
TEXTCAT (macro AUC)   96.39 
SPEED                 19857 


=========================== Textcat F (per label) ===========================

                   P       R       F
datascience    89.29   89.29   89.29
backend        72.73   66.67   69.57
frontend       75.00   69.23   72.00
security       54.55   92.31   68.57
devops         66.67   60.00   63.16
blockchain    100.00   86.67   92.86


======================== Textcat ROC AUC (per label) ========================

              ROC AUC
datascience      0.98
backend          0.92
frontend         0.98
security         0.96
devops           0.95
blockchain       0.99
```

**Comment by davidfischer** [comment]
I also checked and this is an improvement over what is currently in prod.

**Comment by ericholscher** [comment]
Heh, big step up in blockchain performance 🙃

**Pull request merged**